benchmark/buffers/dataview-set.js

```diff
@@ -39,9 +39,9 @@ const mod = {
   setUint32: UINT32
 };
 
-function main({ millions, type }) {
+function main({ n, type }) {
+  const len = n;
```

Comment:
This could just use `n` throughout the `main` function rather than creating an extra variable.

Reply:
It is done to reduce the churn as requested by @BridgeAR

Reply:
No, I actually explicitly excluded the main function in my comment ;-)

Reply:
making the change on the main function too

Reply:
changes made